packages/spark-core/test/integration/spec/spark-core.js

```diff
@@ -26,6 +26,7 @@ describe(`spark-core`, function() {
         })
           .then((res) => {
             assert.property(res.options.headers, `trackingid`);
+            assert.match(res.options.headers.trackingid, /spark-js-sdk_[a-f\d]{8}(?:-[a-f\d]{4}){3}-[a-f\d]{12}_\d+/);
```

Comment:
is there a 'pattern' or 'matchesUtil' file you can move this to? It seems like a regex for a trackingID could be useful in multiple locations.
It is also helpful to include an expected sample that would pass the regex as a comment when dealing with a raw complex expression. eg:
// regex satisfied with spark-js-sdk_12345678-1234-1234-1234-aaaaaaaaaaaa